docs/source-2.0/guides/converting-to-openapi.rst

```diff
@@ -790,6 +790,89 @@ supportNonNumericFloats (``boolean``)
         }
 
 
+.. _generate-openapi-setting-enableDefaultValues:
+
+enableDefaultValues (``boolean``)
```

Comment:
I'd make this `disableDefaultValues` and default it to `false`. booleans with `true` default values are unintutitive.